doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -259,6 +259,7 @@ Bluetooth mesh samples
 * :ref:`bluetooth_mesh_light_lc` sample:
 
   * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+  * Fixed an issue where the sample could return an invalid Light Lightness Status message if the transition time was evaluated to zero.
```

Comment:
Looks good, just consider moving this bullet point before the one that starts with Removed. The order of changes is:
- Added
- Updated
- Fixed
- Removed